components/restapi/core/accounts.go

```diff
@@ -36,6 +31,18 @@ func congestionForAccountID(c echo.Context) (*apimodels.CongestionResponse, erro
 		}
 	}
 
+	hrp := deps.Protocol.CommittedAPI().ProtocolParameters().Bech32HRP()
+	address, err := httpserver.ParseBech32AddressParam(c, hrp, restapipkg.ParameterAddress)
+	if err != nil {
+		return nil, err
+	}
+
+	accountAddress, ok := address.(*iotago.AccountAddress)
+	if !ok {
+		return nil, ierrors.Wrapf(echo.ErrInternalServerError, "failed to assert account address %s", c.Param(restapipkg.ParameterAddress))
```

Comment:
I would say that is not an internal server error, but a user error. The user passed the wrong address type here.
`ErrInvalidParameter`
```suggestion
		return nil, ierrors.Wrapf(httpserver.ErrInvalidParameter, "address %s is not an account address", c.Param(restapipkg.ParameterAddress))
```